chain/rpc/src/helper.rs

```diff
@@ -24,7 +24,7 @@ pub struct Request<'a, T> {
 
 impl<'a, T> Request<'a, T> {
     /// Creates a new JSON RPC request
-    pub fn new(id: u64, method: &'a str, params: T) -> Self {
+    pub fn _new(id: u64, method: &'a str, params: T) -> Self {
```

Comment:
```suggestion
    pub fn new(id: u64, method: &'a str, params: T) -> Self {
```

Reply:
This creates error of not used dead code that cargo bitches about

Reply:
Then why is it here, if it is dead?

Reply:
removed unused code